package.json

```diff
@@ -47,6 +47,7 @@
     "site": "bundle exec jekyll serve --incremental",
     "lint": "tslint -p .",
     "test": "jest test/ && npm run lint && npm run schema && jest examples/ && npm run test:runtime",
+    "test:inspect" :"node --inspect-brk ./node_modules/.bin/jest --runInBand test",
```

Comment:
Doesn't jest have a shortcut for this?

Reply:
I couldn't find anything so I guess this is good.

Reply:
Got this from the [jest docs](https://facebook.github.io/jest/docs/en/troubleshooting.html).